lib/server.js

```diff
@@ -63,32 +63,37 @@ function setup(baseDir, nginxPath, controlEndpoint, listenEndpoint, nginxRoot) {
 
     debug('CMD: %s', cmd.join(' '));
     childProcess.exec(cmd.join(' '), function(err, stdout, stderr) {
-      debug(err);
-      debug('Stdout:\n %s', stdout);
-      debug('Stderr:\n %s', stderr);
+      if (err) debug('error', err);
+      debug('stdout: <\n%s>', stdout);
+      debug('stderr: <\n%s>', stderr);
       cb(err);
     });
   }
   app._nginxCmd = _nginxCmd;
 
   function start(cb) {
+    var controlAddr;
     async.series([
-      app.listen.bind(app),
+      function(done) {
+        app.listen(function() {
+          controlAddr = this.address();
+          debug('control listened on:', this.address());
+          return done();
+        });
+      },
       app._configureNginx.bind(app),
       app._nginxCmd.bind(app, 'start')
     ], function(err) {
       if (err) {
-        if (cb) return cb(err);
-        console.log('Error starting load-balancer: ', err.message);
+        console.log('Error starting: ', err.message);
         throw err;
       }
 
-      app.emit('started');
-      console.log(
-        'StrongLoop Nginx Controller listening at: http://%s:%s',
-        app.get('host'),
-        app.get('port')
-      );
+      if (debug.enabled)
+        app.emit('started'); // Enable Explorer
```

Comment:
Is this OK? I want to avoid

```
Browse your REST API at http://0.0.0.0:4322/explorer
```

being the first and most prominent line of the controller output, if its not necessary, but I've no idea if started does anything else.

I think its OK because of https://github.com/strongloop/strong-nginx-controller/blob/master/lib/boot/explorer.js#L16-L22, but I want to make sure that the event isn't used by loopback for anything else.